go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/zitadel/oidc/v2
 go 1.16
 
 require (
+	github.com/dmarkham/enumer v1.5.7 // indirect
```

Comment:
Ahh, interesting.   I know the answer to why: since we don't have a makefile, I made the enumer `go generate` action installl enumer.   The fix is to have a followup `go generate` command that runs tidy.  I can do that in a separate PR.